sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -144,6 +143,14 @@ public enum ManifestOrderBy
         LastUpdatedOnDescending = 1,
         LastUpdatedOnAscending = 2,
     }
+    public partial class ManifestWriteableProperties
```

Comment:
Will there be other kinds of properties also?

Reply:
Yes, the service recently added a property to the `RepositoryWritableProperties` type, which is why we've separated them out.  They'd like the flexibility to be able to add any properties here over time.

Reply:
Sorry, I should clarify - will there ever be `xxxReadOnlyProperties`? If not, I'm wondering why we added the `Writeable` suffix.

Reply:
Yes, each, xProperties (x=Repository,Manifest,Tag) has a subset of properties that can be written to.  These are the `xWriteableProperties`.